microsoft-edge/visual-studio-code/elements-for-edge.md

```diff
@@ -0,0 +1,70 @@
+---
+description: How to use Elements for Microsoft Edge (Chromium) from VS Code
+title: Debug Microsoft Edge (Chromium) from VS Code
+author: erdraud
+ms.author: erdraud
+ms.date: 08/08/2019
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, elements
+---
+
+# Elements for Edge VS Code extension
+
+By adding the [Elements for Edge](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) VS Code extension, you can use the browser's Elements tool from within Visual Studio Code. By either launching or attaching, the Elements tool will connect to an instance of Microsoft Edge, display the runtime HTML structure, and allow you to alter the layout or fix styling issues.
+
+![GIF of the Elements for Edge VS Code extension at work](./media/elements-for-edge.gif)
+
+## Launching Edge From the Elements tab
+
+![Elements for Edge tab](./media/elements-for-edge-logo.jpg) 
+
+Navigate to Elements (the icon shown above). Next to where it says Elements for Microsoft Edge: Targets, there is a plus sign that will open the browser for your app. If you selected the about:blank option, you will have to navigate to your web app in the browser for it to appear in the Elements panel in VS Code.
```

Comment:
I believe this would be better demonstrated by using a screenshot of VS Code and drawing yellow boxes over what specifically the user should click.

Reply:
We should probably italicize or bold about:blank as well to differentiate it from the surrounding text. Refer to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet for the specific syntax.